src/Views/WhatsNewPage.xaml.cs

```diff
@@ -21,6 +22,12 @@ namespace DevHome.Views;
 
 public sealed partial class WhatsNewPage : Page
 {
+    public Uri DevDrivePageKeyUri { get; private set; } = new ("ms-settings:disksandvolumes");
+
+    public Uri DevDriveLearnMoreLinkUri { get; private set; } = new ("https://go.microsoft.com/fwlink/?linkid=2236041");
+
+    public string DevDriveLinkResourceKey { get; private set; } = "WhatsNewPage_DevDriveCard/Link";

```

Comment:
Nit: I would rename the string ID since it seems it's no longer doing the x:uid magic. That may mess with the localization, tho. But I'm not sure this link needs to be localized?